metadata-ingestion/docs/sources/tableau/tableau_pre.md

```diff
@@ -69,6 +70,40 @@ Lineage is emitted as received from Tableau's metadata API for
 - Custom SQL datasources upstream to Embedded or Published Data Source
 - Tables upstream to Custom SQL Data Source
 
+#### Access Roles
+
+The ingestion can be configured to ingest Tableau permissions as access roles. This enables users to request access to Tableau assets from Datahub.
+The Tableau group permissions are fetched from the Tableau API, optionally transformed to your needs, and then added as roles to the Tableau asset in Datahub. Currently, this is only available for Workbooks.
+
+The configuration in the recipe could look something like this:
+```
+source:
+  type: tableau
+  config:
+    connect_uri: https://tableau.example.com
+
+    access_role_ingestion:
+      enable_workbooks: True
+      role_prefix: "AR-Tableau-"
+      group_substring_start: 29
+      role_description: "IAM role required to access this Tableau asset."
+      displayed_capabilities: ["Read", "Write", "Delete"]
+      request_url: "https://iam.example.com/accessRequest?role=$ROLE_NAME"
+      group_name_pattern:
+        allow: [ "^.*_Consumer$" ]
+
+    username: "${TABLEAU_USER}"
+    password: "${TABLEAU_PASSWORD}"
+
+sink:
+  type: "datahub-rest"
+  config:
+    server: "http://localhost:8080"
+```
+Assuming you have groups in Tableau with names like `AB_XY00-Tableau-Access_A_123_PROJECT_XY_Consumer` or `AB_XY00-Tableau-Access_A_123_PROJECT_XY_Analyst` and corresponding IAM roles like `AR-Tableau-PROJECT_XY_Consumer` or `AR-Tableau-PROJECT_XY_Analyst`.
```

Comment:
Please move this assumption to above the recipe and mention how to map this information in recipe to achieve correct transformation.

Reply:
@sid-acryl I moved it above the recipe and improved the wording. Let me know what you think.